### ISSUE TITLE
How to better find security errata

### ISSUE THREAD

**Issue by jdobes** [RedHatInsights/vmaas#151]
E.g. this is clearly an erratum fixing some CVEs (according to associated BZ names) - https://access.redhat.com/errata/RHSA-2018:0512

But it has empty CVEs field so we effectively ignore this erratum as possible security update.

I think we shouldn't return just errata having any CVE associated. We should return any security errata (even without CVEs associated like in this example) + errata having any CVE associated (to include also regular bugfix errata fixing CVEs).

**Comment by chenders1** [comment]
This one looks like it should have CVE information associated, it just got missed. Do we see other examples in the data?

**Comment by ggainey** [comment]
I was under the impression that we'd already agreed on "All RHSAs *and* any other errata including CVEs". If that's not what we're doing right now, it's a bug.

If an RHSA fixes CVEs, but has no CVE references - yeah, that's prob a bug in the errata-release. Those happen sometimes (alas).

**Comment by chenders1** [comment]
I got this answer from product security. Based on it, I agree. We need to do both.

Asking around, the answer I got (with a caveat that this isn't an official answer to provide to customers -- "I guess that verbiage is still being developed) was that the CVE entries in an erratum are only populated if the CVE is fixed by that erratum on *all* architectures. Since this RHSA only fixed some archs, it doesn't list the relevant CVEs so that it doesn't give the impression that it will fix those CVEs universally. "
